website/pages/en/repl.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+
+const React = require("react");
+
+const styles = [
+  "https://unpkg.com/codemirror@5.30.0/lib/codemirror.css",
+];
+
+const scripts = [
```

Comment:
Do you know if there is a way to define some scripts in `siteConfig.js#scripts` which are only included in this page, instead of doing like this?